### ISSUE TITLE
Client can not connect to server

### ISSUE THREAD

**Issue by darkphan** [Darkosto/SkyFactory4Issues#2]
When connecting to the server you get a "Fatally missing registry entries" [error](https://i.imgur.com/YgznAsZ.png).

In the logs on the client you get the following:

> [13:36:02] [main/INFO] [FML]: Registry SoundEvent: Found a missing id from the world headcrumbs:takeiteasy
> [13:36:02] [main/ERROR] [FML]: Unidentified mapping from registry minecraft:soundevents
> [13:36:02] [main/ERROR] [FML]:     headcrumbs:takeiteasy: 560
> [13:36:02] [Netty Client IO #2/ERROR] [FML]: Network Disconnect: Fatally missing registry entries

After removing headcrumbs, twitchcrumbs and withercrumbs I am successfully able to join the server. I have tested both a Topography world (generator-settings={"Topography-Preset":"Sky Factory IIII"} for those curious) as well as a Compact Sky world (level-type=compactsky)

Server logs (note, contentcreatorintegration is in the pack, however the results are the same without it): 
[2018-09-19-1.log.gz](https://github.com/Darkosto/SkyFactory4Issues/files/2398428/2018-09-19-1.log.gz)

**Comment by Turkey2349** [comment]
Is this server hosted on a Linux machine by chance?

**Comment by darkphan** [comment]
Yes. Although I do think I had the issue when I tried on a server in Windows as well.

The Headcrumbs-1.12.2-2.0.4.15 update does fix the issue.
